include/language/en_us.lang.php

```diff
@@ -3927,8 +3928,8 @@
 
 $app_list_strings['email_settings_opt_in_dom'] = array(
     '' => 'Disabled',
-    'opt-in' => 'Opt In',
-    'confirmed-opt-in' => 'Confirmed Opt In'
+    SugarEmailAddress::COI_STAT_OPT_IN => 'Opt In',
```

Comment:
is this ok for a key part of the language string?
`SugarEmailAddress::COI_STAT_OPT_IN`

Reply:
should be OK, it's a string constant using as an ENUM.

Reply:
Just tested, no problem for translations.
Side note: this is one more array mixed with other string formats for languages.
This include file has so many different language string formats and no rule seems to be implemented. 
It also gives so many problems to translators causing failed language installs on parsing errors!
Its a 3456 lines long so it should be more cleaner and easy to find errors